<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>458</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>596</width>
    <height>458</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>596</width>
    <height>458</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>组件</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>static/images/s.png</normaloff>static/images/s.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>0.900000000000000</double>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>601</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>591</width>
      <height>451</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>请选择需要添加的组件</string>
    </property>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>591</width>
       <height>411</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#groupBox{background-color:rgb(255, 255, 255);}</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>568</width>
        <height>800</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>800</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#scrollAreaWidgetContents_3{background-color:rgb(255, 255, 255);}</string>
      </property>
      <widget class="QPushButton" name="pushButton_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>171</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string>组件</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_20">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>370</y>
         <width>171</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string>组件</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>10</y>
         <width>171</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string>组件</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_14">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>610</y>
         <width>171</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string>组件</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>171</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string>组件</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_17">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>610</y>
         <width>171</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string>组件</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_13">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>490</y>
         <width>171</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string>组件</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_12">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>250</y>
         <width>171</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string>组件</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>490</y>
         <width>171</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string>组件</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_21">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>370</y>
         <width>171</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string>组件</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_18">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>490</y>
         <width>171</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string>组件</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>610</y>
         <width>171</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string>组件</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>171</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string>组件</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_19">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>370</y>
         <width>171</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string>组件</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>130</y>
         <width>171</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string>组件</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>10</y>
         <width>171</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string>组件</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_10">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>250</y>
         <width>171</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string>组件</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_9">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>130</y>
         <width>171</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string>组件</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
